src/components/UI/Navbar.tsx

```diff
@@ -0,0 +1,59 @@
+import { Block } from "baseui/block"
+
+import { styled } from "baseui"
+import { Theme } from "baseui/theme"
+import Icons from "../Icons"
+import { Button } from "baseui/button"
+import { SIZE } from "baseui/input"
+import BaseBtn from "./Common/BaseBtn"
+
+const Container = styled<"div", {}, Theme>("div", ({ $theme }) => ({
+  height: "50px",
+  background: $theme.colors.white,
+  display: "flex",
+  alignItems: "center",
+  flexDirection: "row",
+  boxShadow: "inset 0px -1px 0px #E2E2EA",
+  padding: "10px 20px",
+}))
+
+const Navbar = () => {
+  return (
+    <Container>
+      <Block
+        $style={{
+          display: "flex",
```

Comment:
Remove this style overrides for display flex and use global classes instead. We are using this global classes in PhotAI